src/main/java/FRIDAY/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package FRIDAY;
```

Comment:
I think this is a coding standard violation, as packages are only supposed to use names in lowercase.